File: Backend/DigitalStore.Domain/Entities/EducationalLevel.cs
using DigitalStore.Domain.Common;

namespace DigitalStore.Domain.Entities
{
    public class EducationalLevel : BaseEntity
    {
        public required string Name { get; set; } // e.g., "پایه ششم", "پایه هفتم"
        public string? Description { get; set; }
    }
}

File: education_app/lib/features/profile/domain/entities/educational_level.dart
import 'package:equatable/equatable.dart';

class EducationalLevel extends Equatable {
  final int id;
  final String name;

  const EducationalLevel({
    required this.id,
    required this.name,
  });

  @override
  List<Object?> get props => [id, name];
}
